### PR TITLE
API-6603 additional clipboard patients

### PR DESCRIPTION
**Pull request by jblefkove-va**
https://vajira.max.gov/browse/API-6603

Add two new patients with the these appointments:

ZZTEST,ELEVEN
- 3/31/2021 @ 1700 UTC

ZZTEST,ONE
- 4/16/2021 @ 1630 UTC
- 4/19/2021 @ 1630 UTC
- 4/23/2021 @ 1600 UTC

Resources (plus referenced locations and organizations) are copied from CDW

### CODE REVIEW

**Comment by jblefkove-va** [comment]
```
$ curl -s http://localhost:8090/r4/Patient/1011548191V178564
{
  "resourceType": "Patient",
  "id": "1011548191V178564",
  "extension": [{
      "extension": [{
          "url": "ombCategory",
          "valueCoding": {
            "system": "http://hl7.org/fhir/R4/v3/NullFlavor",
            "code": "UNK",
            "display": "Unknown"
          }
        }, {
          "url": "text",
          "valueString": "Unknown"
        }
      ],
      "url": "https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-race"
    }, {
      "extension": [{
          "url": "ombCategory",
          "valueCoding": {
            "system": "https://www.hl7.org/fhir/us/core/CodeSystem-cdcrec.html",
            "code": "2135-2",
            "display": "Hispanic or Latino"
          }
        }, {
          "url": "text",
          "valueString": "Hispanic or Latino"
        }
      ],
      "url": "https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-ethnicity"
    }, {
      "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-birthsex",
      "valueCode": "M"
    }
  ],
  "identifier": [{
      "use": "usual",
      "type": {
        "coding": [{
            "system": "http://hl7.org/fhir/v2/0203",
            "code": "MR"
          }
        ]
      },
      "system": "http://va.gov/mpi",
      "value": "1011548191V178564",
      "assigner": {
        "display": "Master Patient Index"
      }
    }, {
      "use": "official",
      "type": {
        "coding": [{
            "system": "http://hl7.org/fhir/v2/0203",
            "code": "SB"
          }
        ]
      },
      "system": "http://hl7.org/fhir/sid/us-ssn",
      "value": "000000148",
      "assigner": {
        "display": "United States Social Security Number"
      }
    }
  ],
  "name": [{
      "use": "usual",
      "text": "ZZTEST,ELEVEN",
      "family": "ZZTEST",
      "given": ["ELEVEN"]
    }
  ],
  "telecom": [{
      "system": "phone",
      "value": "5712467552",
      "use": "mobile"
    }, {
      "system": "phone",
      "value": "9097896541",
      "use": "home"
    }
  ],
  "gender": "male",
  "birthDate": "1945-05-05",
  "deceasedBoolean": false,
  "address": [{
      "line": ["1234 TEST"],
      "city": "LAS VEGAS",
      "state": "NEVADA",
      "country": "UNITED STATES"
    }, {
      "line": ["23478 ABC WAY"],
      "city": "RIALTO",
      "state": "CALIFORNIA",
      "country": "UNITED STATES"
    }, {
      "line": ["4758 HOLLOWOOD WAY"],
      "city": "MURRIETA",
      "state": "CALIFORNIA",
      "country": "UNITED STATES"
    }
  ],
  "maritalStatus": {
    "coding": [{
        "system": "http://hl7.org/fhir/marital-status",
        "code": "S",
        "display": "Never Married"
      }
    ],
    "text": "Never Married"
  },
  "contact": [{
      "relationship": [{
          "coding": [{
              "system": "http://hl7.org/fhir/patient-contact-relationship",
              "code": "emergency",
              "display": "Emergency"
            }
          ],
          "text": "Emergency Contact"
        }
      ],
      "name": {
        "text": "LENNON,JOHN"
      },
      "telecom": [{
          "system": "phone",
          "value": "9990001111",
          "use": "home"
        }
      ],
      "address": {
        "line": ["1000 WILSHIRE BLV"],
        "city": "LOS ANGELES",
        "state": "CALIFORNIA"
      }
    }, {
      "relationship": [{
          "coding": [{
              "system": "http://hl7.org/fhir/patient-contact-relationship",
              "code": "family",
              "display": "Family"
            }
          ],
          "text": "Next Of Kin"
        }
      ],
      "name": {
        "text": "DOE,JOHN"
      },
      "telecom": [{
          "system": "phone",
          "value": "111 111 1111",
          "use": "home"
        }
      ],
      "address": {
        "line": ["207 HEAVENLY AVE"],
        "city": "GOOD CITY",
        "state": "CALIFORNIA"
      }
    }, {
      "relationship": [{
          "coding": [{
              "system": "http://hl7.org/fhir/patient-contact-relationship",
              "code": "family",
              "display": "Family"
            }
          ],
          "text": "Secondary Next Of Kin"
        }
      ],
      "name": {
        "text": "FIELDS,SALLY"
      },
      "telecom": [{
          "system": "phone",
          "value": "9095558860",
          "use": "home"
        }
      ],
      "address": {
        "line": ["33 N. Summer Breeze"],
        "city": "dreamville",
        "state": "CANADA"
      }
    }
  ],
  "managingOrganization": {
    "reference": "http://localhost:8090/r4/Organization/I2-N32BW5FRUYHZGS3P6RLVAQG2OPMV6WXT57HAVT3H4Q25YU2ANK2Q0000"
  }
}

$ curl -s http://localhost:8090/r4/Appointment?patient=1011548191V178564
{
  "resourceType": "Bundle",
  "type": "searchset",
  "total": 1,
  "link": [{
      "relation": "first",
      "url": "http://localhost:8090/r4/Appointment?patient=1011548191V178564&_count=30&page=1"
    }, {
      "relation": "self",
      "url": "http://localhost:8090/r4/Appointment?patient=1011548191V178564&_count=30&page=1"
    }, {
      "relation": "last",
      "url": "http://localhost:8090/r4/Appointment?patient=1011548191V178564&_count=30&page=1"
    }
  ],
  "entry": [{
      "fullUrl": "http://localhost:8090/r4/Appointment/I2-FAYWO52C7SH2GAOE3BXRFEMDQ2AKWWTA5OSZAVJWHX4MT65RBRJQ0000",
      "resource": {
        "resourceType": "Appointment",
        "id": "I2-FAYWO52C7SH2GAOE3BXRFEMDQ2AKWWTA5OSZAVJWHX4MT65RBRJQ0000",
        "status": "fulfilled",
        "appointmentType": {
          "coding": [{
              "system": "http://terminology.hl7.org/CodeSystem/v2-0276",
              "display": "REGULAR"
            }
          ],
          "text": "REGULAR"
        },
        "description": "Regular appointment, no special eligibility or type of care being provided.  Routine appointment.",
        "start": "2021-03-31T17:00:00Z",
        "end": "2021-03-31T17:30:00Z",
        "minutesDuration": 30,
        "created": "2021-03-24T00:00:00Z",
        "participant": [{
            "actor": {
              "reference": "http://localhost:8090/r4/Location/I2-V6NXVX5QCTTIQUD7G6WLNSATSM34OAEMNIJUT67HKE44AY7JCS4Q0000",
              "display": "LOM ACC TRAINING CLINIC"
            },
            "status": "accepted"
          }, {
            "actor": {
              "reference": "http://localhost:8090/r4/Patient/1011548191V178564",
              "display": "ZZTEST,ELEVEN"
            },
            "status": "accepted"
          }
        ]
      },
      "search": {
        "mode": "match"
      }
    }
  ]
}
```

**Comment by jblefkove-va** [comment]
```
$ curl -s http://localhost:8090/r4/Patient/1011568236V299349
{
  "resourceType": "Patient",
  "id": "1011568236V299349",
  "extension": [{
      "extension": [{
          "url": "ombCategory",
          "valueCoding": {
            "system": "https://www.hl7.org/fhir/us/core/CodeSystem-cdcrec.html",
            "code": "2076-8",
            "display": "Native Hawaiian or Other Pacific Islander"
          }
        }, {
          "url": "text",
          "valueString": "Native Hawaiian or Other Pacific Islander"
        }
      ],
      "url": "https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-race"
    }, {
      "extension": [{
          "url": "ombCategory",
          "valueCoding": {
            "system": "https://www.hl7.org/fhir/us/core/CodeSystem-cdcrec.html",
            "code": "2186-5",
            "display": "Non Hispanic or Latino"
          }
        }, {
          "url": "text",
          "valueString": "Non Hispanic or Latino"
        }
      ],
      "url": "https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-ethnicity"
    }, {
      "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-birthsex",
      "valueCode": "M"
    }
  ],
  "identifier": [{
      "use": "usual",
      "type": {
        "coding": [{
            "system": "http://hl7.org/fhir/v2/0203",
            "code": "MR"
          }
        ]
      },
      "system": "http://va.gov/mpi",
      "value": "1011568236V299349",
      "assigner": {
        "display": "Master Patient Index"
      }
    }, {
      "use": "official",
      "type": {
        "coding": [{
            "system": "http://hl7.org/fhir/v2/0203",
            "code": "SB"
          }
        ]
      },
      "system": "http://hl7.org/fhir/sid/us-ssn",
      "value": "000000049",
      "assigner": {
        "display": "United States Social Security Number"
      }
    }
  ],
  "name": [{
      "use": "usual",
      "text": "ZZTEST,ONE",
      "family": "ZZTEST",
      "given": ["ONE"]
    }
  ],
  "telecom": [{
      "system": "phone",
      "value": "9094351808",
      "use": "mobile"
    }, {
      "system": "phone",
      "value": "9095555555",
      "use": "home"
    }, {
      "system": "phone",
      "value": "909 555 6789",
      "use": "work"
    }
  ],
  "birthDate": "1949-12-30",
  "deceasedBoolean": false,
  "address": [{
      "line": ["4567 toholue drive"],
      "city": "desert hot rocks",
      "state": "CALIFORNIA",
      "country": "UNITED STATES"
    }, {
      "line": ["54321 PLAYA FIELD BLVD", "APT 25 C"],
      "city": "SN BERNRDNO",
      "state": "CALIFORNIA",
      "country": "UNITED STATES"
    }, {
      "line": ["12345 REDLANDS BLVD"],
      "city": "SAN BERNARDINO",
      "state": "CALIFORNIA",
      "country": "UNITED STATES"
    }
  ],
  "maritalStatus": {
    "coding": [{
        "system": "http://hl7.org/fhir/marital-status",
        "code": "M",
        "display": "Married"
      }
    ],
    "text": "Married"
  },
  "contact": [{
      "relationship": [{
          "coding": [{
              "system": "http://hl7.org/fhir/patient-contact-relationship",
              "code": "family",
              "display": "Family"
            }
          ],
          "text": "Next Of Kin"
        }
      ],
      "name": {
        "text": "TEST,FREDDIE"
      },
      "telecom": [{
          "system": "phone",
          "value": "909 555 1212",
          "use": "home"
        }
      ],
      "address": {
        "line": ["12346 MAIN STREET", "APT 25 C"],
        "city": "RIALTO",
        "state": "CALIFORNIA"
      }
    }, {
      "relationship": [{
          "coding": [{
              "system": "http://hl7.org/fhir/patient-contact-relationship",
              "code": "emergency",
              "display": "Emergency"
            }
          ],
          "text": "Emergency Contact"
        }
      ],
      "name": {
        "text": "TEST,FREDDIE"
      },
      "telecom": [{
          "system": "phone",
          "value": "909 555 1212",
          "use": "home"
        }
      ],
      "address": {
        "line": ["12346 MAIN STREET", "APT 25 C"],
        "city": "RIALTO",
        "state": "CALIFORNIA"
      }
    }
  ],
  "managingOrganization": {
    "reference": "http://localhost:8090/r4/Organization/I2-N32BW5FRUYHZGS3P6RLVAQG2OPMV6WXT57HAVT3H4Q25YU2ANK2Q0000"
  }
}

$ curl -s http://localhost:8090/r4/Appointment?patient=1011568236V299349
{
  "resourceType": "Bundle",
  "type": "searchset",
  "total": 3,
  "link": [{
      "relation": "first",
      "url": "http://localhost:8090/r4/Appointment?patient=1011568236V299349&_count=30&page=1"
    }, {
      "relation": "self",
      "url": "http://localhost:8090/r4/Appointment?patient=1011568236V299349&_count=30&page=1"
    }, {
      "relation": "last",
      "url": "http://localhost:8090/r4/Appointment?patient=1011568236V299349&_count=30&page=1"
    }
  ],
  "entry": [{
      "fullUrl": "http://localhost:8090/r4/Appointment/I2-LF7VR4DAUHVKAES4TLZNLUIIA5TLHC5LGTKNLN5F7TQUA6LG3AAA0000",
      "resource": {
        "resourceType": "Appointment",
        "id": "I2-LF7VR4DAUHVKAES4TLZNLUIIA5TLHC5LGTKNLN5F7TQUA6LG3AAA0000",
        "status": "cancelled",
        "cancelationReason": {
          "coding": [{
              "system": "http://www.va.gov/Terminology/VistADefinedTerms/2.98-16",
              "display": "OTHER"
            }
          ],
          "text": "OTHER"
        },
        "appointmentType": {
          "coding": [{
              "system": "http://terminology.hl7.org/CodeSystem/v2-0276",
              "display": "REGULAR"
            }
          ],
          "text": "REGULAR"
        },
        "description": "Regular appointment, no special eligibility or type of care being provided.  Routine appointment.",
        "start": "2021-04-16T16:30:00Z",
        "end": "2021-04-16T17:00:00Z",
        "minutesDuration": 30,
        "created": "2020-10-09T00:00:00Z",
        "comment": "PID=04/16/21 (MRTC)    RTC 6 MONTHS AM ONLY",
        "participant": [{
            "actor": {
              "reference": "http://localhost:8090/r4/Location/I2-V6NXVX5QCTTIQUD7G6WLNSATSM34OAEMNIJUT67HKE44AY7JCS4Q0000",
              "display": "LOM ACC TRAINING CLINIC"
            },
            "status": "accepted"
          }, {
            "actor": {
              "reference": "http://localhost:8090/r4/Patient/1011568236V299349",
              "display": "ZZTEST,ONE"
            },
            "status": "accepted"
          }
        ]
      },
      "search": {
        "mode": "match"
      }
    }, {
      "fullUrl": "http://localhost:8090/r4/Appointment/I2-XQBUPB3ENV3XWI3D53RUVYRY4ZM4KMHCQACPP2PCL5VLJEZWMZBA0000",
      "resource": {
        "resourceType": "Appointment",
        "id": "I2-XQBUPB3ENV3XWI3D53RUVYRY4ZM4KMHCQACPP2PCL5VLJEZWMZBA0000",
        "status": "cancelled",
        "cancelationReason": {
          "coding": [{
              "system": "http://www.va.gov/Terminology/VistADefinedTerms/2.98-16",
              "display": "OTHER"
            }
          ],
          "text": "OTHER"
        },
        "appointmentType": {
          "coding": [{
              "system": "http://terminology.hl7.org/CodeSystem/v2-0276",
              "display": "REGULAR"
            }
          ],
          "text": "REGULAR"
        },
        "description": "Regular appointment, no special eligibility or type of care being provided.  Routine appointment.",
        "start": "2021-04-19T16:30:00Z",
        "end": "2021-04-19T17:00:00Z",
        "minutesDuration": 30,
        "created": "2020-10-09T00:00:00Z",
        "comment": "PID=04/19/21 (MRTC)    RTC 6 MONTHS AM ONLY",
        "participant": [{
            "actor": {
              "reference": "http://localhost:8090/r4/Location/I2-V6NXVX5QCTTIQUD7G6WLNSATSM34OAEMNIJUT67HKE44AY7JCS4Q0000",
              "display": "LOM ACC TRAINING CLINIC"
            },
            "status": "accepted"
          }, {
            "actor": {
              "reference": "http://localhost:8090/r4/Patient/1011568236V299349",
              "display": "ZZTEST,ONE"
            },
            "status": "accepted"
          }
        ]
      },
      "search": {
        "mode": "match"
      }
    }, {
      "fullUrl": "http://localhost:8090/r4/Appointment/I2-XQBUPB3ENV3XWI3D53RUVYRY4Z355PWGBPACJLUBN4PA5Z3PIIJQ0000",
      "resource": {
        "resourceType": "Appointment",
        "id": "I2-XQBUPB3ENV3XWI3D53RUVYRY4Z355PWGBPACJLUBN4PA5Z3PIIJQ0000",
        "status": "cancelled",
        "cancelationReason": {
          "coding": [{
              "system": "http://www.va.gov/Terminology/VistADefinedTerms/2.98-16",
              "display": "OTHER"
            }
          ],
          "text": "OTHER"
        },
        "appointmentType": {
          "coding": [{
              "system": "http://terminology.hl7.org/CodeSystem/v2-0276",
              "display": "REGULAR"
            }
          ],
          "text": "REGULAR"
        },
        "description": "Regular appointment, no special eligibility or type of care being provided.  Routine appointment.",
        "start": "2021-04-23T16:00:00Z",
        "end": "2021-04-23T16:30:00Z",
        "minutesDuration": 30,
        "created": "2020-10-09T00:00:00Z",
        "comment": "PID=04/23/21 (MRTC)    RTC 6 MONTHS AM ONLY",
        "participant": [{
            "actor": {
              "reference": "http://localhost:8090/r4/Location/I2-V6NXVX5QCTTIQUD7G6WLNSATSM34OAEMNIJUT67HKE44AY7JCS4Q0000",
              "display": "LOM ACC TRAINING CLINIC"
            },
            "status": "accepted"
          }, {
            "actor": {
              "reference": "http://localhost:8090/r4/Patient/1011568236V299349",
              "display": "ZZTEST,ONE"
            },
            "status": "accepted"
          }
        ]
      },
      "search": {
        "mode": "match"
      }
    }
  ]
}
```

**Comment by jblefkove-va** [comment]
Supporting resources:

```
$ curl -s http://localhost:8090/r4/Location/I2-V6NXVX5QCTTIQUD7G6WLNSATSM34OAEMNIJUT67HKE44AY7JCS4Q0000
{
  "resourceType": "Location",
  "id": "I2-V6NXVX5QCTTIQUD7G6WLNSATSM34OAEMNIJUT67HKE44AY7JCS4Q0000",
  "identifier": [{
      "system": "https://api.va.gov/services/fhir/v0/r4/NamingSystem/va-clinic-identifier",
      "value": "vha_605_5483"
    }
  ],
  "status": "active",
  "name": "LOM ACC TRAINING CLINIC",
  "description": "LOM ACC TRAINING CLINIC",
  "mode": "instance",
  "type": [{
      "coding": [{
          "display": "MEDICINE CLINIC"
        }
      ],
      "text": "MEDICINE CLINIC"
    }
  ],
  "telecom": [{
      "system": "phone",
      "value": "909 825 7084"
    }
  ],
  "address": {
    "text": "11201 BENTON ST LOMA LINDA CA 92357-1000",
    "line": ["11201 BENTON ST"],
    "city": "LOMA LINDA",
    "state": "CA",
    "postalCode": "92357-1000"
  },
  "managingOrganization": {
    "reference": "http://localhost:8090/r4/Organization/I2-KSVRNYXSI3NPA5Q2NSLHO6PWK4000000",
    "display": "LOMA LINDA HCS"
  }
}

$ curl -s http://localhost:8090/r4/Organization/I2-KSVRNYXSI3NPA5Q2NSLHO6PWK4000000
{
  "resourceType": "Organization",
  "id": "I2-KSVRNYXSI3NPA5Q2NSLHO6PWK4000000",
  "identifier": [{
      "system": "http://hl7.org/fhir/sid/us-npi",
      "value": "1609827153"
    }, {
      "use": "usual",
      "type": {
        "coding": [{
            "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
            "code": "FI",
            "display": "Facility ID"
          }
        ],
        "text": "Facility ID"
      },
      "system": "https://api.va.gov/services/fhir/v0/r4/NamingSystem/va-facility-identifier",
      "value": "vha_605"
    }
  ],
  "active": true,
  "name": "LOMA LINDA HCS",
  "address": [{
      "text": "11201 BENTON ST 11201 Benton Street LOMA LINDA CA 92357-1000",
      "line": ["11201 BENTON ST", "11201 Benton Street"],
      "city": "LOMA LINDA",
      "state": "CA",
      "postalCode": "92357-1000"
    }
  ]
}

$ curl -s http://localhost:8090/r4/Organization/I2-N32BW5FRUYHZGS3P6RLVAQG2OPMV6WXT57HAVT3H4Q25YU2ANK2Q0000
{
  "resourceType": "Organization",
  "id": "I2-KSVRNYXSI3NPA5Q2NSLHO6PWK4000000",
  "identifier": [{
      "system": "http://hl7.org/fhir/sid/us-npi",
      "value": "1609827153"
    }, {
      "use": "usual",
      "type": {
        "coding": [{
            "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
            "code": "FI",
            "display": "Facility ID"
          }
        ],
        "text": "Facility ID"
      },
      "system": "https://api.va.gov/services/fhir/v0/r4/NamingSystem/va-facility-identifier",
      "value": "vha_605"
    }
  ],
  "active": true,
  "name": "LOMA LINDA HCS",
  "address": [{
      "text": "11201 BENTON ST 11201 Benton Street LOMA LINDA CA 92357-1000",
      "line": ["11201 BENTON ST", "11201 Benton Street"],
      "city": "LOMA LINDA",
      "state": "CA",
      "postalCode": "92357-1000"
    }
  ]
}
```

File: minimart-manager/src/test/java/gov/va/api/health/minimartmanager/minimart/MitreMinimartMaker.java [hunk 1]
```diff
@@ -160,20 +160,21 @@
               .build();
 
   private final Function<DatamartOrganization, OrganizationEntity> toOrganizationEntity =
-      (dm) -> {
-        var maybeFacilityId = dm.facilityId().orElse(FacilityId.builder().build());
+      dm -> {
+        var address =
+            Optional.ofNullable(dm.address())
+                .orElse(DatamartOrganization.Address.builder().build());
```
Comment (jblefkove-va) [inline minimart-manager/src/test/java/gov/va/api/health/minimartmanager/minimart/MitreMinimartMaker.java:166]: One of the test organizations does not populate address

**Review by bschofield-va — APPROVED** [review]

**Pull request merged**